src/server/templates/base.html

```diff
@@ -50,12 +50,21 @@
     <header>
       <div class="container">
         <div class="row d-flex align-items-center">
-          <h1 class="col-3 flex-grow-1">Epidemic Forecasting</h1>
+          <h1 class="col-3 flex-grow-1"><a href="/">Epidemic Forecasting</a></h1>
           <nav class="col nav nav-masthead justify-content-end">
-            <a class="nav-link active" href="/">Home</a>
-            <a class="nav-link" href="/model">Model</a>
-            <a class="nav-link" href="/request-calculation">Request model</a>
-            <a class="nav-link" href="/request-event-evaluation">Events Prediction</a>
+            {% set navigation_bar = [ ('/', 'index', 'Public information'),
```

Comment:
formatting by prettier for these is terrible

Reply:
Maybe we could set longer lines?
https://github.com/prettier/prettier/issues/5751

"prettier.printWidth": 150
https://prettier.io/docs/en/configuration.html#basic-configuration

Reply:
I think it would be still ugly, this hopefully won't change that much.

Reply:
OK?